src/TTextEdit.h

```diff
@@ -157,11 +161,13 @@ private slots:
     TConsole* mpConsole;
     QPointer<Host> mpHost;
     QScrollBar* mpScrollBar;
+    QScrollBar* mpHScrollBar;
```

Comment:
This pointer is declared here and initialised (to a `nullptr`) in the constructor - but it is not used - the control concerned is created in the `TConsole` constructor and all references to it are via the same named pointer in the `public` part of that other class.

tl;dr; this is not needed HERE or in the `TTextEdit` constructor...

Reply:
Thanks for noticing that. I noticed that it is the same for the pointer above so I got rid of that one as well.